scripts/dockerhost/start-dockerhosts

```diff
@@ -80,25 +97,30 @@ first="true"
 for i in `seq 1 $num_nodes`; 
 do
     host="netplugin-node$i"
-    echo "Starting $host"
-    # godep modifies the host's GOPATH env variable, CONTIV_HOST_GOPATH
-    # contains the unmodified path passed from the Makefile, use that
-    # when it is defined.
-    if [ -n $CONTIV_HOST_GOPATH ]; then
```

Comment:
I think this change is needed to correctly mount the host's gopath.

The current sanity failure seems unrelated though.